xdsl/utils/test_value.py

```diff
@@ -1,9 +1,7 @@
-from typing import Generic
+from xdsl.dialects.test import TestOp
+from xdsl.ir import AttributeCovT, OpResult
 
-from xdsl.ir import AttributeCovT, Block, Operation, SSAValue
 
-
-class TestSSAValue(Generic[AttributeCovT], SSAValue[AttributeCovT]):
-    @property
-    def owner(self) -> Operation | Block:
-        raise ValueError("Attempting to get the owner of a `TestSSAValue`")
+def TestSSAValue(t: AttributeCovT) -> OpResult[AttributeCovT]:
```

Comment:
I get minimising the diff, but CamelCase for a function feels wrong

Reply:
Yeah I was thinking of moving it over either in this PR or a subsequent one if everyone was happy with the mechanism. @math-fehr ?

Reply:
I'm otherwise happy with the idea of "testSSAValue" (or similar name) just being a helper to create a test op

Reply:
I'm fine in keeping it for a bit, but yeah we should change it to `get_test_ssa_value` or something.